espflash/tests/README.md

```diff
@@ -1,6 +1,6 @@
 # Test Resources
 
-This document describes how the test files under `tests/resources` were generated, so that they can be re-generated in the future if needed.
+This document describes how the test files under `tests/data` were generated, so that they can be re-generated in the future if needed.
```

Comment:
As a note, this document needs updating. However, we no longer have a `blinky` example in the HAL, so have just left it as-is for now :)

Reply:
> As a note, this document needs updating

Shall we open an issue about this?